tests/stdlib/test_math.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+
+from ..utils import TranspileTestCase
+
+
+class MathModuleTests(TranspileTestCase):
```

Comment:
Maybe add some more edge case:
- Where the exponent is negative.
- Where both the base and the exponent is negative.
- When the output is a complex number.